db/schema.rb

```diff
@@ -78,16 +79,6 @@
     t.index ["content_id"], name: "index_link_sets_on_content_id", unique: true, using: :btree
   end
 
-  create_table "linkables", force: :cascade do |t|
-    t.integer  "content_item_id", null: false
-    t.string   "state",           null: false
-    t.string   "base_path",       null: false
-    t.string   "document_type",   null: false
-    t.datetime "created_at"
-    t.datetime "updated_at"
-    t.index ["content_id"], name: "index_link_sets_on_content_id", unique: true, using: :btree
-  end
-
```

Comment:
You could squash this in to the previous commit, for neatness (but I'll leave it up to you).